modules/config/URLProcessor.js

```diff
@@ -1,66 +1,76 @@
-/* global config, interfaceConfig, loggingConfig, getConfigParamsFromUrl */
+/* global config, interfaceConfig, loggingConfig */
+
 const logger = require("jitsi-meet-logger").getLogger(__filename);
 
-var configUtils = require('./Util');
-var params = {};
+import { getConfigParamsFromUrl } from '../../react/features/base/util';
+
+import configUtils from './Util';
 
-params = getConfigParamsFromUrl();
+const URL_PARAMS = getConfigParamsFromUrl({ url: window.location });
 
-var URLProcessor = {
-    setConfigParametersFromUrl: function () {
-        // Convert 'params' to JSON object
-        // We have:
-        // {
-        //   "config.disableAudioLevels": false,
-        //   "config.channelLastN": -1,
-        //   "interfaceConfig.APP_NAME": "Jitsi Meet"
-        // }
-        // We want to have:
-        // {
-        //   "config": {
-        //     "disableAudioLevels": false,
-        //     "channelLastN": -1
-        //   },
-        //   interfaceConfig: {
-        //     APP_NAME: "Jitsi Meet"
-        //   }
-        // }
-        var configJSON = {
+/**
+ * Convert 'URL_PARAMS' to JSON object
+ * We have:
+ * {
+ *      "config.disableAudioLevels": false,
+ *      "config.channelLastN": -1,
+ *      "interfaceConfig.APP_NAME": "Jitsi Meet"
+ * }
+ * We want to have:
+ * {
+ *      "config": {
+ *          "disableAudioLevels": false,
+ *          "channelLastN": -1
+ *      },
+ *      interfaceConfig: {
+ *          "APP_NAME": "Jitsi Meet"
+ *      }
+ * }
+ */
+export default {
+    setConfigParametersFromUrl() {
+        const configJSON = {
             config: {},
             interfaceConfig: {},
             loggingConfig: {}
         };
-        for (var key in params) {
-            if (typeof key !== "string") {
-                logger.warn("Invalid config key: ", key);
-                continue;
+
+        Object.keys(URL_PARAMS).forEach(key => {
```

Comment:
Please undo these changes in the first commit, since you're doing them again in the 3rd.